templates/ModelDecoderEncoderSnippet.stencil

```diff
@@ -18,4 +23,9 @@ public func encode(to encoder: Encoder) throws {
     {% for property in model.properties %}
     {% include "ModelEncoderProperty.stencil" %}
     {% endfor %}
+    {% for constant in model.constants %}
+    {% if constant.required == true %}
+    try container.encode({{ constant.name }}, forKey: .{{ constant.name }})
+    {% endif %}
+    {% endfor %}
```

Comment:
I guess I don't understand--if the swagger defines a constant, when would it ever not be sent over the wire? Looking at this, I would think all constants would be required and am not sure what role a non-required constant would play or why it would be in Swagger.

Reply:
Yes. you are right, the check is not needed. I removed the 'required' check in sending Constants in http request. Stencil files and ConstantViewModel are also updated. Thanks @tjprescott